### PR TITLE
PROD-28 : Created basic web interface

### PR DESCRIPTION
**Pull request by EnricoTrentin**
I created a few controllers and jsp pages in the webapp module.

### CODE REVIEW

File: webapp/src/main/webapp/WEB-INF/jsp/about.jsp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          <a class="nodecoration" href="/about">About</a>
+          <a class="nodecoration" href="/credits">Credits</a>
+        </nav>
+
+        <section id="watchtower">
+          <h3>About Watchtower</h3>
+          <div class="textblock">
+              <p>Visualise your software metrics easily and quickly with watchtower.</p>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+              <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (ThomasPerkins1123) [inline webapp/src/main/webapp/WEB-INF/jsp/about.jsp:34]: I am assuming all of this is just sample text that we probably don't need anymore?

File: webapp/src/main/webapp/WEB-INF/jsp/error.jsp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<html lang="en">
+<head>
+
```
Comment (ThomasPerkins1123) [inline webapp/src/main/webapp/WEB-INF/jsp/error.jsp:6]: Do we maybe want to add at least one thing in this head such as a title?

**Review by ThomasPerkins1123 — APPROVED** [review]
It looks as if your git user is not set up correct on the machine you are using?
Not critical but will look nicer if the git contributors e.t.c shows your actual account

File: webapp/src/main/webapp/WEB-INF/jsp/about.jsp (hunk 1 above)
Comment (EnricoTrentin) [reply webapp/src/main/webapp/WEB-INF/jsp/about.jsp]: true!

File: webapp/src/main/webapp/WEB-INF/jsp/error.jsp (hunk 2 above)
Comment (EnricoTrentin) [reply webapp/src/main/webapp/WEB-INF/jsp/error.jsp:9]: I added it in a new commit

**Pull request merged**